docs/source/quickstart-aws.rst

```diff
@@ -194,15 +194,15 @@ It is also recommended each time you start a new ElasticBLAST search.
 
 The delete command will take a few minutes to run as it needs to manage multiple cloud resources.
 
-You may verify that your cloud resources have been deleted by running: 
+After the ``elastic-blast delete`` command returns, you may verify that your
+cloud resources have been deleted by running the command below. Its output will
+show the EC2 instance IDs ``elastic-blast`` created on your behalf that are
+still in the ``running`` state.
```

Comment:
"Its output will
show the EC2 instance IDs ``elastic-blast`` created on your behalf that are
still in the ``running`` state."

I do not think this is true if multiple users run elastic-blast in cloud shell, like in the testing session.

Reply:
That is true, but for any upcoming such testing sessions, I intend to remind everyone of the tip below (provided belatedly at the testing session). My guess is that for most users, this won't be the case.

> Here is a belated tip for those working with ElasticBLAST on the AWS Cloud Shell that will facilitate tracking your individual searches: If you set the USER environment variable to your NCBI login, this will be reflected in the AWS resources that elastic-blast creates on your behalf (e.g.: AWS CloudFormation stack, AWS Batch resources, EC2 instances, but not your results bucket). Here is an example:
export USER=<YOUR_NCBI_LOGIN_HERE>

Reply:
It will be the case if they run elastic-blast in cloud-shell. They may not know about each other and they will see the other's running instances that we tell then to interpret as theirs.

Reply:
On further thought, this will actually not be the case: each user _will_ see a different set of EC2 instances, as their `$USER` environment variable will be set to something along the lines of `cloudshell-user-77ce53a49` (see evidence from screenshots below). Hopefully this  `$USER` value doesn't get recycled each time they refresh their browser when working on the AWS CloudShell! 

<img width="2048" alt="Screen Shot 2021-04-01 at 4 15 08 PM" src="https://user-images.githubusercontent.com/736979/113453763-ac6b6000-93d4-11eb-8485-80f854229f10.png">

Reply:
> It will be the case if they run elastic-blast in cloud-shell.

That is correct, hence the (poor man's) workaround:
`export USER=<YOUR_NCBI_LOGIN_HERE>`

Reply:
If we had an easy way to compute the cluster name (e.g.: `elasticblast-camacho-84152930b`), then this could be used instead of the `Owner` tag to identify the instances properly.